### DISCUSSION TITLE
Welcome to Developer Support!

### DISCUSSION THREAD

**Discussion by OPMattie** [ethereum-optimism/developers#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sbvegan** [comment]
Hello everyone!

I'm soyboy and I am a developer support engineer at OPLabs! I was drawn to the Optimism Collective because of two primary reasons:

1. Retroactive Public Goods Funding seems like an excellent way to fund open source software and I think it can be applied beyond software to allow underfunded positive sum human resources to flourish. I'm hoping that it fosters a strong Ethereum development ecosystem and it becomes a case study for other applications.
2. I think scaling decentralized compute is important and doing it with a MIT License is incredibly based. While I appreciate the beauty of the cathedral, I'd rather have fun in a bustling bazaar. 

I still have a lot to learn and contribute. So I look forward to doing it alongside other developers on every layer of the OP Stack.

**Comment by opfocus** [comment]
Hello, I'm simple8720, and I come from the Optimism Discord community. I'm a member of TechNerds, and in the recent past, I've been proud to assist in resolving some fundamental questions about using components related to Optimism.

I have a keen interest in public goods and align closely with the vision and values of Optimism. I'm genuinely curious about the transformative potential of decentralized technologies in our society. While the journey may seem complex at times, I remain optimistic and patient about the positive impact it can bring.

What I find particularly fascinating is the opportunity to interact closely with both users and developers. It's a dynamic experience that I find immensely enjoyable.

I look forward to engaging further with the community!

**Reply by sbvegan** [reply]
I'm so glad to have you as a member our community simple! You've been an immense help to the developers in the Optimism Collective. Your hard work answering questions in Discord is always noticed and appreciated :)

**Comment by joohhnnn** [comment]
Hello everyone!

I'm joohhnnn, a blockchain engineer from EPF4. I have a keen interest in Ethereum ecosystem technologies such as geth and opstack, and I'm very open to sharing and exchanging knowledge with anyone who shares similar interests. If you have questions regarding the codebase or some lower-level issues, feel free to reach out to me anytime. Additionally, if you are looking to develop some valuable public goods and need assistance, I'd be more than happy to help. I believe that through collective wisdom and experience sharing, we can together advance the blockchain technology. Looking forward to learning and growing with all of you in this community!

**Reply by sbvegan** [reply]
joohhnnn! Happy to have you here, thanks for being a part of our community :)
